src/components/usa-breadcrumb/usa-breadcrumb.stories.js

```diff
@@ -0,0 +1,24 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-breadcrumb.twig";
+import Data from "./usa-breadcrumb.json";
+import WrapData from "./usa-breadcrumb--wrap.json";
```

Comment:
Should we standardize file naming to use the ~ instead of -- to indicate variant?

Reply:
I could be wrong but if you look at the table component, it looks like `--` represents a variant of a component and `~` represents a _modifier_ 

@mejiaj  thoughts?

Reply:
We should move away from using `~` in the future. This was a pattern lab style.